pkg/config/ip.go

```diff
@@ -143,3 +149,49 @@ func GetExternalIP(stunServers []string, localAddr net.Addr) (string, error) {
 		}
 	}
 }
+
+func validateExternalIP(ctx context.Context, nodeIP string, addr *net.UDPAddr) error {
```

Comment:
comments on this function would be helpful

Reply:
Done